src/main/java/org/codehaus/plexus/components/io/attributes/SimpleResourceAttributes.java

```diff
@@ -177,9 +192,10 @@ public PlexusIoResourceAttributes setOctalModeString( String mode )
         return this;
     }
 
+    @Override
     public String toString()
     {
-        return String.format(
+        return String.format( Locale.ENGLISH,
```

Comment:
Why English locale?

Reply:
Mostly because the message is English, yet I will change it to the ROOT locale as it makes no difference in this case.